Shell/LineEditor.cpp

```diff
@@ -226,6 +226,23 @@ String LineEditor::get_line(const String& prompt)
                 fflush(stdout);
                 continue;
             }
+            if (ch == 0x01) // ^A
+            {
+                if (m_cursor > 0) {
+                    fprintf(stdout, "\033[%dD", m_cursor);
```

Comment:
```suggestion
                    printf("\033[%dD", m_cursor);
```

`fprintf(stdout)` is really just `printf()`